### ISSUE TITLE
"Call requires the user to be admin of the ad account" on admin user

### ISSUE THREAD

**Issue by doubtx** [facebook/facebook-python-ads-sdk#289]
Hello, I am trying to query Facebook API to get Ad campaigns data, it worked fine couple days ago, but now it returns error: This Ads API call requires the user to be admin of the ad account. User is not admin on ad account

Here is the code in Python

> 

```
from facebookads import FacebookAdsApi
from facebookads import objects

import json
import os

class FacebookStats():
    """
    Class for fetching FB stats
    """
    def __init__(self):
        token_file = open(os.path.join(os.path.dirname(__file__), 'config/fb_token.json'))
        token_details = json.load(token_file)
        token_file.close()

        # Initialize Facebook connector
        FacebookAdsApi.init(**token_details)

        # Load config file
        config_file = open(os.path.join(os.path.dirname(__file__), 'config/fb_details.json'))
        config = json.load(config_file)
        config_file.close()

        self.account = objects.AdAccount(config['ad_account'])

    def get_stats(self):

        return self.account.get_insights(fields=['impressions', ])

fb = FacebookStats()

stats = fb.get_stats()
```

>

**Comment by JiamingFB** [comment]
Hi @doubtx,

This is likely a permission issue or token issue.

First of all, could you please try to login as the user, and then go to Facebook Ads Manager or Power Editor to check if the user still has permissions on the account?

If you can still see and edit the ad account, then we need some private info (you user ID, account ID, access token, etc.) for debugging. Could you please report this issue at https://developers.facebook.com/bugs/ instead, so that we can securely get those private info? 

Thanks!

**Comment by doubtx** [comment]
Done, thank you!

**Comment by cristiangrama** [comment]
It will have been nice to report what was the issue for others.

**Comment by doubtx** [comment]
Hey, sorry.
The problem was that there was no active billing method configured for account, that resulted in not really correct error response from API

**Comment by benoitdemaegdt** [comment]
Hey !
Just to let you know that I faced the same issue.
I was getting Ad campaigns data from Marketing API for quite a while when I suddenly got this error message :
`This Ads API call requires the user to be admin of the ad account. User is not admin on ad account XXXX`
I found out that it was a credit card issue. It seems that when Facebook is not anymore able to debit your credit card, you get this error message.

Hope it can help someone !
